sdk/src/main/java/io/dapr/config/Properties.java

```diff
@@ -189,10 +191,17 @@ public Properties() {
 
   /**
    * Creates a new instance to handle Properties per instance.
-   * @param overrides to override static properties
-   */
-  public Properties(Map<String, String> overrides) {
-    this.overrides = Collections.unmodifiableMap(overrides);
+   * @param overridesInput to override static properties
+   */
+  public Properties(Map<?, String> overridesInput) {

```

Comment:
Here we can also use `Map.of()` and `Map.copyOf()` so the expression below becomes:
```
this.overrides = overridesInput == null ? Map.of() :
        Map.copyOf(overridesInput.entrySet().stream()
            .filter(e -> e.getKey() != null)
            .filter(e -> e.getValue() != null)
            .collect(Collectors.toMap(
                entry -> entry.getKey().toString(),
                entry -> entry.getValue()
            )));
```